packages/components/package.json

```diff
@@ -54,45 +54,44 @@
     "clsx": "^1.1.1"
   },
   "devDependencies": {
-    "@babel/cli": "^7.14.8",
-    "@babel/core": "^7.15.0",
-    "@babel/plugin-transform-modules-commonjs": "^7.15.0",
-    "@babel/preset-env": "^7.15.0",
+    "@babel/cli": "^7.15.7",
+    "@babel/core": "^7.15.8",
+    "@babel/plugin-transform-modules-commonjs": "^7.15.4",
+    "@babel/preset-env": "^7.15.8",
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.15.0",
-    "@chanzuckerberg/axe-storybook-testing": "^4.1.2",
+    "@chanzuckerberg/axe-storybook-testing": "^4.1.3",
     "@chanzuckerberg/story-utils": "^2.1.0",
     "@percy/storybook": "^3.3.1",
-    "@storybook/addon-a11y": "^6.3.8",
-    "@storybook/addon-essentials": "^6.3.8",
+    "@storybook/addon-a11y": "^6.3.12",
+    "@storybook/addon-essentials": "^6.3.12",
     "@storybook/addon-postcss": "^2.0.0",
-    "@storybook/addons": "^6.3.8",
     "@storybook/preset-typescript": "^3.0.0",
-    "@storybook/react": "^6.3.8",
+    "@storybook/react": "^6.3.12",
     "@storybook/storybook-deployer": "^2.8.10",
     "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@types/jest": "^26.0.24",
+    "@testing-library/react": "^12.1.2",
+    "@types/jest": "^27.0.2",
     "@types/react": "^17.0.16",
-    "autoprefixer": "^10.3.1",
-    "axe-core": "^4.3.2",
-    "babel-jest": "^27.0.6",
-    "babel-loader": "^8.2.2",
-    "chromatic": "^5.10.1",
+    "autoprefixer": "^10.4.0",
+    "axe-core": "^4.3.5",
+    "babel-jest": "^27.3.1",
+    "babel-loader": "^8.2.3",
+    "chromatic": "^6.0.5",
+    "classnames": "^2.3.1",
```

Comment:
We're using classnames in a story, so it's in dev deps here. If we ever want to use it in an actual component, we'll need to move it to regular deps.

Reply:
Wait, just realized we're using clsx also... will change this and the story to that